hashdiff.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |s|
 
   s.add_development_dependency('bluecloth')
   s.add_development_dependency('rspec', '~> 2.0')
-  s.add_development_dependency('rubocop')
+  s.add_development_dependency('rubocop', '~> 0.80.0')
```

Comment:
needs to be nailed down or new PRs randomly fail on new cops

Reply:
Thanks @grosser. The error seems to not related to this. I don't know how to fix `rubocop` errors, could you please advise?

Reply:
ah yeah ... I guess new version of rubocop not working on old ruby versions :(
... would you accept dropping all "EOL" ruby versions from dependencies ?
(they can still install hashdiff, will just get an older version)
... then we can also use frozen strings/keywords etc modern features 🤷‍♂

Reply:
... otherwise we should drop to this rubocop https://rubygems.org/gems/rubocop/versions/0.49.1 to have consistent cops